tests/Settings.ts

```diff
@@ -14,8 +14,8 @@ export class Settings {
     // Endpoint should reduce standard speech timeout to value specified in SpeechServiceTimeoutSeconds
     // If undefined, production timeout of 10 seconds will be used, but at the cost of greatly incrased test
     // duration.
-    public static SpeechTimeoutEndpoint: string;
-    public static SpeechTimeoutKey: string;
+    public static SpeechTimeoutEndpoint: string = "wss://fedev-reconnect.develop2.cris.ai/speech/recognition/conversation/cognitiveservices/v1?language=en-us";
+    public static SpeechTimeoutKey: string = "c42994717bb44e9b843188ab0ad2d7b6";
```

Comment:
> public static SpeechTimeoutKey: string = "c42994717bb44e9b843188ab0ad2d7b6"; [](start = 3, length = 77)

(just curious) do we want this kind of information in the public repo?

Reply:
Gah! No.

Reply:
Keys were regenerated.